source/images/download-full.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="44px" height="44px" viewBox="0 0 44 44" version="1.1" xmlns="http://www.w3.org/2000/svg">
+    <title>Download</title>
```

Comment:
This isn't translatable, whereas the alt attribute text on the element calling this in is translatable. Will need to check with a screenreader whether having this here makes things better or worse for a language other than English.

Reply:
Need to use `xml:lang` at least.